### PR TITLE
create initial fruit tables

### PR DESCRIPTION
**Pull request by mcfiredrill**
https://docs.google.com/document/d/1xdZ1pxNuVfyqQEKxl5Cz6UG5haxEJp-fi-U5NU1bvq0/edit?usp=sharing

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -117,6 +137,18 @@
     t.index ["user_id"], name: "index_microtexts_on_user_id", using: :btree
   end
 
+  create_table "patreons", force: :cascade do |t|
```
Comment (mcfiredrill) [inline db/schema.rb:140]: This shouldnt be here :x:

**Comment by rbxbx** [comment]
are the integer cost fields going to be stored in cents? i think that's usually encouraged to avoid eg: rounding errors and such -- I've used https://github.com/RubyMoney/money-rails in quite a few projects to fair success that adds some nice abstractions when dealing with currencies 

just some initial thoughts!

**Comment by mcfiredrill** [comment]
> are the integer cost fields going to be stored in cents
@rbxbx 
Yeah it's been awhile since I've done anything like this but I believe I've stored cents as an integer before and it worked out nicely.

And good tip about ruby-money that's a great library.

**Comment by mcfiredrill** [comment]
Thinking of closing this in favor of https://github.com/datafruits/streampusher-api/pull/160

**Pull request closed without merging**